src/components/ScriptureCard.js

```diff
@@ -1,56 +1,65 @@
-import { useEffect } from 'react'
+import React from 'react';
 import PropTypes from 'prop-types'
-import {
-  Card,
-  CardContent,
-  useContent,
-  useCardState,
-} from 'translation-helps-rcl'
-import { ScripturePane, useScripture } from 'single-scripture-rcl'
+import { Card, useCardState } from 'translation-helps-rcl'
+import { ScripturePane } from 'single-scripture-rcl'
 import { getLanguage } from '@common/languages'
+import { ComboBox } from '@components/ComboBox';
+import { getItemByTitle, updateTitle } from '@utils/ScriptureVersionHistory';
+import { getScriptureVersionSettings, useScriptureSettings } from '@hooks/useScriptureSettings';
 
-export default function ScriptureCard({
-  title,
-  chapter,
-  verse,
-  server,
-  owner,
-  branch,
-  languageId,
-  classes,
-  bookId,
-  resourceId,
-  disableWordPopover,
-}) {
-  const scriptureConfig = useScripture({
-    reference: {
-      projectId: bookId,
-      chapter,
-      verse,
-    },
-    resource: {
-      languageId,
-      projectId: resourceId,
-      owner,
-      branch,
-    },
-    config: {
-      server,
-      cache: { maxAge: 1 * 1 * 1 * 60 * 1000 },
-    },
-    disableWordPopover,
-  })
+const label = 'Version';
+const style = { marginTop: '16px', width: '500px' };
+
+export default function ScriptureCard(Props) {
+  const {
+    title,
+    classes,
+  } = Props;
+
+  const { scriptureConfig, setScripture } = useScriptureSettings(Props);
+
+  if (scriptureConfig.title) {
+    const title = `${scriptureConfig.title} v${scriptureConfig.version}`
+    updateTitle(scriptureConfig.resourceLink, title);
+  }
 
-  const language = getLanguage({ languageId })
-  const direction = language?.direction || 'ltr'
+  function getDropDownComponent() {
```

Comment:
Why isn't the DropDownComponent part of the translation-helps-rcl instead of the create-app codebase?

Reply:
It started out there, but it became so specialized to scripture resources and it wasn't even needed by the t-helps-rcl.  So I changed the settings card in the t-helps-rcl to pass in a generic method to add a custom component into the settings.  It should probably be renamed since this is no longer a generic dropdown.  I need to think about other options.

Reply:
Should it be part of the single-scripture-rcl then? I'm thinking it should be part of an RCL so other apps can use it as well if they find it useful.

Reply:
certainly need to be more consistent with naming - call it either a drop down or comboBox.